types/openlayers/openlayers-tests.ts

```diff
@@ -1111,6 +1112,35 @@ styleRegularShape = new ol.style.RegularShape({
     points: 4,
 });
 
+//
+// ol.style.Stroke
+//
+
+styleStroke = new ol.style.Stroke();
+styleStroke.setColor('#FF0000');
+styleStroke.setColor('red');
+styleStroke.setColor('#CCC');
+styleStroke.setColor('rgb(255, 255, 255)');
+styleStroke.setColor('rgb(255, 255, 255, 0.7)');
+styleStroke.setLineCap('butt');
+styleStroke.setLineCap('round');
+styleStroke.setLineCap('square');
+styleStroke.setLineJoin('bevel');
+styleStroke.setLineJoin('round');
+styleStroke.setLineJoin('miter');
+styleStroke.setLineDash([10, 5]);
+styleStroke.setLineDashOffset(10);
+styleStroke.setMiterLimit(20);
+styleStroke.setWidth(5);
+
+styleStroke.getColor();
```

Comment:
For the getters, you should affect the returned value to typed variables.

Reply:
Did so with latest commit.